integration/test/Test/NginxZAuthModule.hs

```diff
@@ -0,0 +1,181 @@
+module Test.NginxZAuthModule where
```

Comment:
`Test.Nginz.ZAuthModule`?  Since we already have `Test.Nginz`...  (Or just add the former to the latter?)